src/templates/classic/components/Image/Image.tsx

```diff
@@ -1,12 +1,10 @@
-import { HTMLAttributeAnchorTarget } from "react"
 import { ImageProps } from "~/common"
 
 const Image = ({ src, alt, width }: ImageProps) => (
-  <img src={src} alt={alt} width={`${width ?? 100}%`} />
+  <img src={src} alt={alt} width={`${width ?? 100}%`} height="auto" />
 )
 
-
-const ImageComponent = ({ src, alt, width, href }: ImageProps) => {
+const ImageComponent = ({ src, alt, width, href, type }: ImageProps) => {
```

Comment:
I don't think we need to destructure `type`, it's only used by the render engine when deciding what component to render

Reply:
hmm its just a typing thing at the moment, since the props for Image are `ImageProps`, in which `type` is required. 

type error noticed in main line today:
![Screenshot 2024-03-21 at 8 18 05 AM](https://github.com/isomerpages/isomer-components/assets/42832651/008e6fff-0c03-421a-8201-f4cdc776eaf1)

Reply:
wah actually got quite a number of errors, opening #40 to make this more obvious to devs. after fix lets make it a requirement to passing ci before merge to main line!

Reply:
ah ok, yea should probably use`Omit<ComponentProps, "type">` for the actual component props
the `type` is only needed by render engine

can see Next `Paragraph` for reference

Reply:
ya thats better, changed